pom.xml

```diff
@@ -11,20 +11,21 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>oboe.js</artifactId>
-    <version>2.0.3-SNAPSHOT</version>
+    <version>2.1.3-SNAPSHOT</version>
     <name>Oboe.js</name>
     <description>WebJar for Oboe.js</description>
     <url>http://webjars.org</url>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <upstream.version>2.0.2</upstream.version>
-        <upstream.url>https://raw.githubusercontent.com/jimhigson/oboe.js/v${upstream.version}/dist</upstream.url>
+        <upstream.version>2.1.3</upstream.version>
+        <upstream.url>https://cdn.rawgit.com/jimhigson/oboe.js/v${upstream.version}/dist</upstream.url>
         <destDir>${project.build.outputDirectory}/META-INF/resources/webjars/${project.artifactId}/${project.version}</destDir>
         <requirejs>
             {
                 "paths": {
-                    "oboe-js": "oboe-browser.js"
+                    "oboe.js": "oboe-browser.js",
+                    "oboe.js": "oboe-browser.min.js"
```

Comment:
ok, I had a poor understanding of the logic behind paths, I'll make a new pull request as soon as I'll  have understood everything.